api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -205,6 +205,10 @@ message Principal {
     type.matcher.v3.StringMatcher principal_name = 2;
   }
 
+  reserved 5;
+
+  reserved "source_ip";
```

Comment:
This is removed due to a tooling issue that @htuch is working on; he says it should be sorted out within a week, but we'll need to wait to merge this until the tooling is fixed, because we can't remove the field.

Reply:
Yeah, please go ahead and get this review otherwise finalized, I'm hoping to land the required PRs this week.

Reply:
Thanks, I updated the generated proto manually for the code review. I will run `fix_format` later when the tooling issue is fixed.